bootstrap

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+autoscan
```

Comment:
That would be because I removed all files automatically generated by autotools from the version control, more specifically those created by the first execution of both: 'libtoolize' (ltmain.sh) and 'automake --add-missing --force-missing' (ar-lib, compile, config.guess, config.sub, depcomp, install-sh, missing, test-driver). Running autoreconf will fail, unless you run these bootstrap commands in that particular order.

I did not know that users should be able to build the project directly from the repository without having to install GNU auto tools, thus I removed all automatically generated files. That's something that I have already seen before in some projects, e.g.: wayland and weston.

I was hoping that we could use 'make dist' in order to create the distributable tar ball, which does contain configure, Makefile.in, etc.